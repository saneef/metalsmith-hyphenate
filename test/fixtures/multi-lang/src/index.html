<blockquote>Die Weltgeschichte ist auch die Summe dessen, was vermeidbar gewesen w√§re.<cite>Konrad Adenauer (1876-1967), West Germany's first chancellor</cite></blockquote>
<blockquote lang="en-us">An armed society is a polite society. Manners are good when one may have to back up his acts with his life.<cite>Robert A. Heinlein</cite></blockquote>