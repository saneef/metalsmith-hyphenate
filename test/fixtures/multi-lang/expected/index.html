<html><head></head><body><blockquote>Die Welt­ge­schich­te ist auch die Sum­me des­sen, was ver­meid­bar ge­we­sen wäre.<cite>Kon­rad Ade­nau­er (1876-1967), West Ger­many's first chan­cel­lor</cite></blockquote>
<blockquote lang="en-us">An ar­med so­cie­ty is a po­li­te so­cie­ty. Man­ners are good when one may have to back up his acts with his life.<cite>Ro­bert A. Hein­lein</cite></blockquote></body></html>